v2/gcs-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/transforms/GcsToSourceStreamer.java

```diff
@@ -74,6 +93,7 @@ public void setup() {
     while (retry) {
       try {
         spannerDao = new SpannerDao(spannerConfig, tableSuffix, isMetadataDbPostgres);
+        storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
         retry = false;
       } catch (SpannerException e) {
         LOG.info("Exception in setup of AssignShardIdFn {}", e.getMessage());
```

Comment:
Need to change the exception message , AssignShardIdFn is not the right name

Reply:
done